.github/renovate.json5

```diff
@@ -0,0 +1,30 @@
+{
+  "extends": [
+    "github>newrelic/coreint-automation:renovate-base.json5"
```

Comment:
do the automerge rules apply to the `regexManagers` defined here?

Reply:
I've updated it here: [3b6b5fd](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/187/commits/3b6b5fdbbda10580b24040ada1200a763600c7f4)

Regarding the automerge, renovate opens PRs for these changes. [example](https://github.com/sigilioso/newrelic-prometheus-exporters-packages/pull/8)

<img width="968" alt="image" src="https://github.com/newrelic/newrelic-prometheus-exporters-packages/assets/442627/875c42fc-50d9-43f3-8f95-8398dfcd4eb1">